server/api/orders.js

```diff
@@ -0,0 +1,37 @@
+const router = require('express').Router()
+const { Order } = require('../db/models')
+module.exports = router
+
+
+// GET /api/orders
+router.get('/', (req, res, next) => {
+  Order.findAll({})
+    .then(orders => res.json(orders))
+    .catch(next)
+})
+
+// GET /api/orders/:status
+router.get('/:status', (req, res, next) => {
+  Order.findAll({
+    where: {
+      status: req.params.status
+    }
+  })
+    .then(orders => res.json(orders))
+    .catch(next)
+});
+
+// GET /api/orders/:orderId
+router.get('/:orderId', (req, res, next) => {
+  Order.findById(req.params.orderId)
+    .then(order => res.json(order))
+    .catch(next)
+});
+
+// PUT /api/orders/:orderId
+router.put('/:orderId', (req, res, next) => {
+  Order.findById(req.params.orderId)
+  .then(order => order.update(req.body))
+    .then(newOrder => res.status(201).json(newOrder))
```

Comment:
What do you believe the value of `newOrder` is, right here? I’m thinking it’s not what you think it is…

Reply:
No, I know what it is. I meant to look up the syntax for returning.  Is it just {returning: true}?

Reply:
Yes, and then on the Exp-Seq checkpoint you also used `plain: true`…